client/src/assets/sharedComponents/DetectFormData.tsx

```diff
@@ -0,0 +1,10 @@
+function DetectFormData(formData: object) {
+  for (const key in formData) {
+    if ((formData as any)[key] !== '') {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
would something a little simpler and more readable work
like:
`Object.values(formData).filter(val => Boolean(val)).length > 0`
haha better readability is debatable now that i've written it out
i'll still leave it here and leave it up to you

Reply:
yeah it's going to look weird both ways